src/Fantomas/CodePrinter.fs

```diff
@@ -1703,9 +1703,16 @@ and genExpr astContext synExpr ctx =
                 genExpr astContext e
                 +> sepSpace
                 +> col sepSpace es (fun (s, oe, e) ->
+                    let isLambda =
```

Comment:
I think we can re-use

https://github.com/fsprojects/fantomas/blob/04d176a519106751750e29e5754075eb475e827c/src/Fantomas/SourceParser.fs#L1798-L1802

here.

Reply:
Thanks for pointing that out.